modules/viz/include/opencv2/viz/types.hpp

```diff
@@ -127,11 +133,17 @@ namespace cv
 
             Mat texture, tcoords;
 
-            /** @brief Loads a mesh from a ply file.
+            /** @brief Loads a mesh from a ply or a obj file.
+
+            @param file File name
+            @param type File type (for now only PLY and OBJ are supported)
 
-            @param file File name (for now only PLY is supported)
+            **File type** can be one of the following:
+            -   **LOAD_PLY**
+            -   **LOAD_OBJ**
              */
-            static Mesh load(const String& file);
+            static Mesh load(const String& file, int type = LOAD_PLY);
```

Comment:
For me it's ok. I don't know if this is allowed or not. Perhaps should ask @vpisarev